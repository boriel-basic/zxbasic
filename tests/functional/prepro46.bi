#define X(a) a

#if X(1)
print 1
#endif



