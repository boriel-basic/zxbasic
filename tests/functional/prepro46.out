#line 1 "prepro46.bi"



print 1
#line 6 "prepro46.bi"



